changelogs/unreleased/gh-6200-lua_cjson-encode-escapes-slash.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/lua/json
```

Comment:
In the PR description:

> NO_DOC=[docs on tarantool.wiki](https://github.com/Totktonada/test-wiki-subpages/wiki/compat%3Ajson_escape_forward_slash)

The link should be https://github.com/tarantool/tarantool/wiki/compat%3Ajson_escape_forward_slash, I guess.

Reply:
Fixed.